e2e/validators.go

```diff
@@ -30,30 +30,30 @@ func ValidateSysctlConfig(ctx context.Context, s *Scenario, customSysctls map[st
 	for k := range customSysctls {
 		keysToCheck = append(keysToCheck, k)
 	}
-	execResult := execOnVMForScenarioValidateExitCode(ctx, s, fmt.Sprintf("sysctl %s | sed -E 's/([0-9])\\s+([0-9])/\\1 \\2/g'", strings.Join(keysToCheck, " ")), 0, "systmctl command failed")
+	execResult := execOnVMForScenarioValidateExitCode(ctx, s, fmt.Sprintf("sudo sysctl %s | sed -E 's/([0-9])\\s+([0-9])/\\1 \\2/g'", strings.Join(keysToCheck, " ")), 0, "systmctl command failed")
 	for name, value := range customSysctls {
 		require.Contains(s.T, execResult.stdout.String(), fmt.Sprintf("%s = %v", name, value), "expected to find %s set to %v, but was not", name, value)
 	}
 }
 
 func ValidateNvidiaSMINotInstalled(ctx context.Context, s *Scenario) {
-	command := "nvidia-smi"
+	command := "sudo nvidia-smi"
 	execResult := execOnVMForScenarioValidateExitCode(ctx, s, command, 1, "")
 	require.Contains(s.T, execResult.stderr.String(), "nvidia-smi: command not found", "expected stderr to contain 'nvidia-smi: command not found', but got %q", execResult.stderr.String())
 }
 
 func ValidateNvidiaSMIInstalled(ctx context.Context, s *Scenario) {
-	command := "nvidia-smi"
+	command := "sudo nvidia-smi"
 	execOnVMForScenarioValidateExitCode(ctx, s, command, 0, "could not execute nvidia-smi command")
 }
 
 func ValidateNvidiaModProbeInstalled(ctx context.Context, s *Scenario) {
-	command := "nvidia-modprobe"
+	command := "sudo nvidia-modprobe"
```

Comment:
I think sudo is fine in testing contexts, but why exactly do we need to add it now?